app/grandchallenge/algorithms/tasks.py

```diff
@@ -330,7 +259,6 @@ def create_algorithm_jobs(
     *,
     algorithm_image,
     civ_sets,
-    creator=None,
```

Comment:
It is clear that this is only for creating anonymous jobs, but maybe -for dummies- that could be explicitly said in the name of the task or a short docstring?